semantic/src/site/globals/site.overrides

```diff
@@ -1835,6 +1835,10 @@ a:hover {
 #data-view-config-panel {
   padding-top: 20px;
   width: 50%;
+  @media only screen and (max-width: @largestMobileScreen) {
+    width: unset;
+    max-width: 100%;
+  }
```

Comment:
This just makes the Data Views and System Preferences config pages appear slightly better on mobile. Not strictly necessary, but was simple enough.